rpc/core/types/responses.go

```diff
@@ -128,17 +128,23 @@ type ResultValidators struct {
 }
 
 // Info about the consensus state.
-// Unstable
+// UNSTABLE
 type ResultDumpConsensusState struct {
 	RoundState json.RawMessage `json:"round_state"`
 	Peers      []PeerStateInfo `json:"peers"`
 }
 
+// UNSTABLE
```

Comment:
Is this tagging meant to indicate API stability?

Reply:
yes: https://github.com/tendermint/tendermint#semver